### PR TITLE
Illustrator Test Scripts added

### PR DESCRIPTION
**Pull request by PavanKumarN8**
Hi @sigadamvenkata, I have added Illustrator Test Scripts.

Local Test Run Results:
image (1)

### CODE REVIEW

**Review by Dli3 — CHANGES_REQUESTED** [review]
You need to sign the Adobe CLA.

File: selectors/feds/feds.illustratorsanity.page.js [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+          if (locale === 'Germany') {
+            await expect(element).toBeVisible();
+          }
+          break;
+        default:
+          await expect(element).toBeVisible();
+          break;
+      }
+    });
+    await Promise.all(promises);
```
Comment (Dli3) [inline selectors/feds/feds.illustratorsanity.page.js:130]: Do you need this?

File: selectors/feds/feds.illustratorsanity.page.js [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+          await expect(element).toBeVisible();
+          break;
+        case this.adobeFireflyMp:
+          if (locale === 'Poland') { await expect(element).toBeVisible(); }
+          break;
+        default: await expect(element).toBeVisible();
+      }
+    });
+    await Promise.all(promises);
+    await this.creativityAndDesign.click({ timeout: 5000 });
```
Comment (Dli3) [inline selectors/feds/feds.illustratorsanity.page.js:168]: You use '.click({ timeout: 5000})' a lot in your PRs. If the default isn't meeting your needs, you can change the default in your team's config. See https://playwright.dev/docs/test-timeouts .

File: selectors/feds/feds.illustratorsanity.page.js [hunk 3]
```diff
@@ -0,0 +1,271 @@
...
+      }
+    });
+    await Promise.all(promises);
+  }
+
+  // Test Script
+  async validatingIllustratorPage(page, baseURL, featureIndex, locale, illustrator) {
+    console.info(`[FEDSInfo] Checking Page: ${baseURL}${features[featureIndex].path}`);
+
+    await test.step(`Validating ${locale} Locale Page`, async () => {
```
Comment (Dli3) [inline selectors/feds/feds.illustratorsanity.page.js:252]: Why isn't this in your test file? You can probably break this down to multiple steps there.

**Comment by PavanKumarN8** [comment]
I will be raising the new PR with changes as discussed. Hence closing this.

**Pull request closed without merging**